### PR TITLE
Add "get-scitokens-mapfile" script that turns Credentials.TokenIssuers data into a condor mapfile of "SCITOKEN" lines

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: bin/get-scitokens-mapfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        if is_null(vo_data, "Credentials", "TokenIssuers"):
+            continue
+        mapfile += f"# {vo_name}\n"
+        for token_issuer in vo_data["Credentials"]["TokenIssuers"]:
+            url = token_issuer.get("URL")
+            unix_user = token_issuer.get("DefaultUnixUser")
+            if url and unix_user:
+                mapfile += f"SCITOKEN {url} {unix_user}\n"
+            else:
+                mapfile += f"# invalid SCITOKEN: {url or '<NO URL>'} {unix_user or '<NO UNIX USER>'}\n"
```
Comment (brianhlin) [inline bin/get-scitokens-mapfile:49]: Cool, I didn't know you could do that with fstrings. Do these comments get printed for VOs without `Credentials.TokenIssuers`?

File: bin/get-scitokens-mapfile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    all_vos_data = webapp.models.GlobalData().get_vos_data()
+    for vo_name, vo_data in all_vos_data.vos.items():
+        if is_null(vo_data, "Credentials", "TokenIssuers"):
+            continue
+        mapfile += f"# {vo_name}\n"
+        for token_issuer in vo_data["Credentials"]["TokenIssuers"]:
+            url = token_issuer.get("URL")
+            unix_user = token_issuer.get("DefaultUnixUser")
+            if url and unix_user:
+                mapfile += f"SCITOKEN {url} {unix_user}\n"
```
Comment (brianhlin) [inline bin/get-scitokens-mapfile:47]:
```suggestion
                mapfile += f"SCITOKEN \"{url}\" {unix_user}\n"
```

**Review by brianhlin — APPROVED** [review]
A few comments but otherwise LGTM. Pre-approving.

File: bin/get-scitokens-mapfile (hunk 1 above)
Comment (matyasselmeci) [reply bin/get-scitokens-mapfile:49]: Nope, the `is_null()` above takes care of skipping those. And yeah, fstrings are great.

File: bin/get-scitokens-mapfile (hunk 2 above)
Comment (matyasselmeci) [reply bin/get-scitokens-mapfile:49]: Did it a different way so I wouldn't have to change the string in the error case as well.

**Pull request merged**